doc/src/manual/functions.md

```diff
@@ -993,6 +993,9 @@ julia> f.(A, B)
  33.0
 ```
 
+Keyword arguments are not broadcasted over, but are simply passed through to each call of
+the function.  For example, `round.(x, digits=3)` is equivalent to `broadcast(x -> round(x, digits=3), x)`.
```

Comment:
`;` instead of `,`? It makes clearer the difference between keyword and positional arguments, and also then you can use the implicit kwarg naming

Reply:
I don't want to give the impression that you need a `;` for it to not be broadcasted.

Isn't `digits=3` already clearly a keyword argument?  What else could it be?

Reply:
In python, it can still be a positional argument. I always use semicolons in my code (it's clearer, and more consistent with the `f(; x)` synonym for `f(; x=x)`), and I see https://github.com/invenia/BlueStyle#keyword-arguments does it too, but in the base docs it says "When the function is called, the semicolon is optional: one can either call plot(x, y, width=2) or plot(x, y; width=2), but the former style is more common", so apparently it's not as clear-cut as I thought, forget I said anything!